src/plugins/kibana/public/settings/sections/indices/add_data_steps/pipeline_setup/lib/__tests__/pipeline.js

```diff
@@ -415,11 +421,11 @@ describe('processor pipeline', function () {
       expect(pipeline.output).to.be(expected);
     });
 
-    it('should set output to be undefined if no processors exist', function () {
+    it('should set output to be equal to input if no processors exist', function () {
       const pipeline = new Pipeline();
 
       pipeline.updateOutput();
-      expect(pipeline.output).to.be(undefined);
+      expect(pipeline.output).to.be(pipeline.input);
```

Comment:
To make this test more robust I think you should explicitly set pipeline.input before calling updateOutput(). As it stands, this test would pass if both input and output were just initialized to undefined.